tests/TypeReconciliation/EmptyTest.php

```diff
@@ -473,19 +473,6 @@ function foo(array $r) {
                     }',
                 'error_message' => 'MixedReturnTypeCoercion'
             ],
-            'secondEmptyTwice' => [
```

Comment:
why did you drop this? Seems like it still works on my side

Reply:
I migrated the test (in the more common falsiness check form) to `RedundantConditionTest.php`